### DISCUSSION TITLE
How do you compile your own runners?

### DISCUSSION THREAD

**Discussion by roadapathy** [lutris/lutris#5652] [Q&A] [answered]
**This is a burning question of mine: How do you compile a runner?**

I compile Valve's **Proton** all the time in hopes of adding a slight performance bump. I wrote a guide and put that on Reddit.

I also compile the **Linux Kernel** which does add a noticeable performance bump- especially with newer Kernels with updated AMDGPU drivers for my AMD GPU. 

However, I have no idea how to do this. I set my Lutris game to use the Steam Proton that I compiled but it doesn't work. There are too many modifications to the runner specifically for Star Citizen and therefore my mainline Proton didn't work.

I know, this is a big question but I'd really like to know, please.

**Comment by strycore** [comment, marked as answer]
Every Lutris runner is built from scripts present in the buildbot repository: https://github.com/lutris/buildbot/

We don't use the Wine build script anymore and use Wine-GE / Proton-GE which you will find build instructions in their respective repos.

Other than that, you're going to have to publish actual benchmarks and numbers for your claims of custom compiling makes things faster.
